hokusai/production.yml

```diff
@@ -133,3 +133,34 @@ spec:
             backend:
               serviceName: team-web-internal
               servicePort: 3000
+---
```

Comment:
I shamelessly stole this from [horizon](https://github.com/artsy/horizon/blob/f210c4c595519229fa1099eb456bbcb8531d2574/hokusai/production.yml#L156-L186)